app/controllers/Application.scala

```diff
@@ -34,18 +34,27 @@ class Application @Inject() (
   val messagesApi: MessagesApi,
   val env:Environment[User,CookieAuthenticator],
   configuration: Configuration,
+  dispenserService: DispenserService,
   socialProviderRegistry: SocialProviderRegistry) extends Silhouette[User,CookieAuthenticator] {
 
   val pool1Export = configuration.getBoolean("pool1.export").getOrElse(false)
 
   def index = SecuredAction(Pool1Restriction(pool1Export)).async { implicit request =>
-    Future.successful(Ok(views.html.index(request.identity, request.authenticator.loginInfo)))
+    val template: Template = dispenserService.buildTemplate(
```

Comment:
Yes it is to much overhead for a simple rendering, but i think the http filter are not exactly what we need.  So I add a improvement for this issues https://github.com/Viva-con-Agua/drops/issues/211 .
It will work with a special rendering extended the Ok() Function